### DISCUSSION TITLE
Grohe Blue Home integration in Home Assistant

### DISCUSSION THREAD

**Discussion by koendruyts** [lennartkaden/Grohe-Blue-API-Python#2] [Ideas]
Dear,

I was looking to get my Grohe Blue Home integrated into Home Assistant and the closed integration I could find on Github was this one. 

Any idea how to get all data out from the Grohe Watersystems app?
https://compact-training.cloud.grohe.com/en/blue-home/watersystems-app/index.html
e.g. track water consumption would be a nice feature to include in my general water consumption.

Thx

**Comment by lennartkaden** [comment]
Hi @koendruyts,

that is indeed a good idea for a future extension, which I will definitely keep an eye on.

I myself am currently having difficulties with my device hardware, so I can't currently experiment on extensions to the API. However, as soon as that becomes a possibility, I will look into it.

Beyond that, feel free to create a fork and experiment on it yourself. 

Greetings
